internal/provider/provider.go

```diff
@@ -39,9 +40,17 @@ func New(version string) func() *schema.Provider {
 		p := &schema.Provider{
 			Schema: map[string]*schema.Schema{
 				"host": {
-					Type:        schema.TypeString,
-					Required:    true,
-					DefaultFunc: schema.EnvDefaultFunc("DOCKER_HOST", "unix:///var/run/docker.sock"),
+					Type:     schema.TypeString,
+					Required: true,
+					DefaultFunc: func() (interface{}, error) {
+						if v := os.Getenv("DOCKER_HOST"); v != "" {
+							return v, nil
+						}
+						if runtime.GOOS == "windows" {
+							return "npipe:////.//pipe//docker_engine", nil
```

Comment:
Asking because I don't own a windows: I double-checked with https://docs.docker.com/desktop/faqs/general/ and there the value is `npipe:////./pipe/docker_engine`, only one `/` between `pipe` and `docker_engine`.
Which is the correct value?

Reply:
Fixed! It seems to work with both.